Dockerfile

```diff
@@ -95,7 +95,7 @@ RUN pip install -r /doc-requirements.txt
 RUN if [ "$MIOTENSILE_VER" = "latest" ] ; then cget -p $PREFIX install ROCmSoftwarePlatform/MIOpenTensile@7568654c938d42e9a91c6b18fb382f5b978d12fd; else cget -p $PREFIX install ROCmSoftwarePlatform/MIOpenTensile@5fe0bf4a8dc59f3ab62df929297280915372ce16; fi
 
 RUN cd ~ && \
-    export MLIR_COMMIT=06a6d3fc1af1fdfbe706dbfcd588fb3744526124 && \
+    export MLIR_COMMIT=31d92f4c64ae6fa6b7c5d543f68b69300b4513ce && \
```

Comment:
I think this would break all other solvers. You need to disable those solvers and respective tests, otherwise CI tests won't pass.

Reply:
Could you clarify what this means?

I think I should update the Jenkinsfile's default docker image name because I updated the dockerfile. Will do that in next commit. But I'm not sure why bumping the commit id will break all other solvers or CI?

Reply:
Should I bump this to `miopen-v1`?

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8e2f418d41e456f760ef7deb61d4244635ce0818/Jenkinsfile#L67

Reply:
> Should I bump this to `miopen-v1`?

This is not needed anymore, see #801

Reply:
> Could you clarify what this means?

Unmodified MLIR solvers for Bwd/WrW directions do not pass num CUs and device name to miirBuildViaHip.

Reply:
@atamazov Thanks for link to #801.

> Unmodified MLIR solvers for Bwd/WrW directions do not pass num CUs and device name to miirBuildViaHip.

You reminded me, thank you! Locally I have already changed both bwd and wrw solvers that's why I didn't see any issue coming up. Now, it just look like it doesn't worth it to hold back those changes because it will make the code base dependent on different MLIR commit. I will add the other two changes to the PR two, but in different commit, strictly similar to changes I made in fwd so hopefully this doesn't make your review to be a pain.

Reply:
Gotcha!

But okay. Just use different commits. Sometimes keeping this kind of discipline is painful, but it is worth it.

Reply:
[Note] Each commit pushed to repo will start a new CI job. So make all your commits locally and then push once.

Reply:
@atamazov 

> But okay. Just use different commits. Sometimes keeping this kind of discipline is painful, but it is worth it.

Next time I should adjust my development model to this (Not just myself ahead of the PR) as well. This time reverting the changes + disabling the solvers does not worth it considering I have the changes ready.

> [Note] Each commit pushed to repo will start a new CI job. So make all your commits locally and then push once.

I noticed that, but thanks for reminding.